spec/models/user_spec.rb

```diff
@@ -8,5 +8,14 @@
     it { is_expected.to validate_presence_of(:email) }
     it { is_expected.to validate_uniqueness_of(:name) }
     it { is_expected.to validate_uniqueness_of(:email) }
+    it { is_expected.to have_secure_password }
+  end
+
+  context "admin" do
+    it "is false by default" do
+      user = create(:user)
```

Comment:
No reason to create a record in the database for this spec. Better to use `User.new`
And works really nicely with the spec magic matchers: `expect(User.new).not_to be_admin`